portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +29,7 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  ArrayList<String> comments = new ArrayList<String>();
```

Comment:
I think the convention should be
```
List<String> comments = new ArrayList<String>();
```

Reply:
https://stackoverflow.com/questions/147468/why-should-the-interface-for-a-java-class-be-preferred